### PR TITLE
Corrections and new language

### PR DESCRIPTION
**Pull request by tihhgoncalves**
**Correction:**
I was not carrying the translation skills correctly.

**New language:**
Added language Portuguese (Brazil)

Note: I did not know if I could add my name as a "_contributor_", but I would like to be part of your team.

**Hugs! :)**

### CODE REVIEW

**Comment by tihhgoncalves** [comment]
I added improvements in the Portuguese (Brazil) translation and another correction.

File: medium.php [hunk 1]
```diff
@@ -31,3 +31,10 @@
   require_once(MEDIUM_PLUGIN_DIR . "lib/medium-site.php");
   add_action("init", array("Medium_Site", "init"));
 }
+
+/* Load translations (textdomain) */
+add_action( 'init', 'medium_load_textdomain' );
```
Comment (majelbstoat) [inline medium.php]: Why does this need to be in init, and why can't it be in admin_init?

File: languages/medium-pt_BR.po [hunk 2]
```diff
@@ -0,0 +1,433 @@
...
+
+#: lib/medium-admin.php:60
+msgid ""
+"An updated integration token is needed to perform this action. Please create "
+"a new integration token from your Medium settings page and set it on your "
+"WordPress profile above."
+msgstr ""
+"Um token de integração atualizado é necessário para executar esta ação. Por "
+"favor, crie um novo token de integração de sua página de configurações "
+"médias e configurá-lo no seu perfil de WordPress acima."
```
Comment (cassiosantos) [inline languages/medium-pt_BR.po:26]: Acredito que "página de configurações médias" deveria ser "página de configurações do Medium"

File: languages/medium-pt_BR.po [hunk 3]
```diff
@@ -0,0 +1,433 @@
...
+msgid "%s (Draft only)"
+msgstr "%s (somente rascunho)"
+
+#: lib/medium-admin.php:1080
+msgid "Invalid method specified."
+msgstr "Método especificado inválido."
+
+#: lib/medium-admin.php:1106
+msgid "Unexpected response format: %s - %s"
+msgstr "Resposta inesperada: %s - %s"
```
Comment (cassiosantos) [inline languages/medium-pt_BR.po:102]: Acho que fica melhor "formato de resposta inesperado".

File: languages/medium-pt_BR.po [hunk 4]
```diff
@@ -0,0 +1,433 @@
...
+msgid "Migrate to Medium"
+msgstr "Migrar para Medium"
+
+#: views/form-migrate-tool.phtml:3
+msgid ""
+"Use this migration tool to cross-post all posts on this site to a Medium "
+"publication."
+msgstr ""
+"Utilize esta ferramenta de migração para publicar todas as publicações deste "
+"site numa publicação Meium."
```
Comment (cassiosantos) [inline languages/medium-pt_BR.po:131]: "Meium" faltou um `d`

File: languages/medium-pt_BR.po [hunk 5]
```diff
@@ -0,0 +1,433 @@
...
+msgid ""
+"Migrated posts will not be distributed to users, and will use their original "
+"post date as the post date on Medium."
+msgstr ""
+"Publicações migradas não serão distribuídas aos usuários e usarão a data "
+"original de publicação como a data de postagem no Medium."
+
+#: views/form-migrate-tool.phtml:5
+msgid "Migrated posts will not be cross-linked on either site."
+msgstr "As postagens migradas não serão re-linkadas em nenhum dos sites."
```
Comment (cassiosantos) [inline languages/medium-pt_BR.po:143]: Seria interessante padronizar para "Publicações migradas" (como utilizado na linha 138)

**Comment by cassiosantos** [comment]
@tihhgoncalves and @majelbstoat  

I just added some considerations about the translation into Brazilian Portuguese. 

It depends on small adjustments but the translation looks good!

**Pull request closed without merging**